hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/condition/Alert.java

```diff
@@ -89,6 +91,22 @@
     @JsonInclude
     private String resolvedNotes;
 
+    /*
+     * This is the trigger defined when the alert was fired.
+     * A trigger definition can change during time, but an alert should be attached with a specific instance.
+     */
+    @JsonInclude(Include.NON_EMPTY)
+    @Thin
+    private Trigger trigger;
+
+    /*
+     * This is the dampening attached to a trigger when the alert was fired.
+     * As a trigger, the dampening can change during time, but an alert should be attached with a specific instance.
+     */
+    @JsonInclude(Include.NON_EMPTY)
+    @Thin
+    private Dampening dampening;
+
     @JsonInclude(Include.NON_EMPTY)
```

Comment:
I'm not totally convinced we need to drag around this information on an Alert, it makes a heavy object even heavier and it could increase the complexity of converting from json back to pojo if there are changes to the objects.  But I guess I'm not so opposed as to call for the removal, I'm just not sure it's a good idea.